app/models/statistics_announcement.rb

```diff
@@ -207,8 +207,24 @@ def requires_redirect?
     unpublished? || publication_has_been_published?
   end
 
+  def can_be_tagged_to_taxonomy?
+    organisations_content_ids = organisations.map(&:content_id)
+
+    organisations_in_tagging_beta?(organisations_content_ids)
+  end
 
 private
+
+  def organisations_in_tagging_beta?(org_content_ids)
+    return false if org_content_ids.empty?
+
+    organisations_in_tagging_beta = Whitehall.organisations_in_tagging_beta
+
+    org_content_ids.any? do |id|
```

Comment:
This runs at  O(n<sup>2</sup>) so maybe very slow if either of the arrays is sizeable. 

Could it work with an intersection instead? `(organisations_in_tagging_beta & org_content_ids).present?`